interfaces/cython/cantera/examples/onedim/premixed_counterflow_twin_flame.py

```diff
@@ -87,6 +87,9 @@ def solveOpposedFlame(oppFlame, massFlux=0.12, loglevel=1,
 # Create the flame object
 oppFlame = ct.CounterflowTwinPremixedFlame(gas, width=width)
 
+# Use mixture-averaged properties
+oppFlame.transport_model = 'Mix'
```

Comment:
Just to confirm, this line isn't necessary if you're using the now-updated master branch, correct?